### PR TITLE
Angular 11 upgrade

### PR DESCRIPTION
**Pull request by lukestanley**
I have managed to manually upgrade to Angular 11.
The image-tagger now builds with the latest version of @angular-devkit/build-ng-packagr, which is depreciated, and should probably be sorted in the future.

Needs special testing:
- [ ] - JSON Schema validator
- [ ] - LeafletModule
- [ ] - LeafletMarkerClusterModule (these are both for maps)
- [ ] - auth0
- [ ] - image-tagger at /kendraio/photoTagger

Todo:

- [x] - address insecure package warnings and minor upgrades
- [x] - update develop to Angular 10 to make diff more small

### CODE REVIEW

**Comment by the-greenman** [comment]
Have merged latest dev changes into this branch.

File: src/app/blocks/csv-import-block/csv-import-block.component.ts [hunk 1]
```diff
@@ -49,7 +50,7 @@ export class CsvImportBlockComponent implements OnInit, OnChanges {
     const fileReader = new FileReader();
     fileReader.onload = (_) => {
       const { type, ...config } = this.config;
-      const parsedData = parse(fileReader.result as any, config);
+      const parsedData = parse(fileReader.result as string, config as ParseConfig) as ParseResult<any>;
```
[the remark this answers is not in the document]
Comment (the-greenman) [reply src/app/blocks/csv-import-block/csv-import-block.component.ts:53]: All my imports have worked so far.

File: src/app/blocks/validate-block/validate-block.component.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ export class ValidateBlockComponent extends BaseBlockComponent {
     //  Types of property 'opts' are incompatible.
     //    The types returned by 'compare(...)' are incompatible between these types.
     //                           Type 'boolean' is not assignable to type 'number'.
-    addFormats(ajv as any);
+    addFormats(ajv);
```
Comment (the-greenman) [inline src/app/blocks/validate-block/validate-block.component.ts:41]: The validate block is hard to test ... I don't even know if it works.

**Review by the-greenman — APPROVED** [review]
All tests on key changes here have been good so far.

File: src/app/blocks/csv-import-block/csv-import-block.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukestanley) [reply src/app/blocks/csv-import-block/csv-import-block.component.ts:53]: I don't expect the 'as' type information to effect anything at run time. @the-greenman

[the remark this answers is not in the document]
Comment (lukestanley) [reply src/app/blocks/csv-import-block/csv-import-block.component.ts:53]: I did this so it could compile.

**Pull request merged**